### DISCUSSION TITLE
Pull request (PR) view does not show recent commits

### DISCUSSION THREAD

**Discussion by denproc** [community/community#59774] [Pull Requests]
### Select Topic Area

Bug

### Body

Hi,

I have successfully pushed the changes into the [remote brach](https://github.com/photosynthesis-team/piq/tree/docs/fill_gaps). However, the [PR view](https://github.com/photosynthesis-team/piq/pull/368/commits) doesn't show the recent commit. I'm not sure about the cause for such behaviour. 

Screenshots:
Branch View vs PR view
![image](https://github.com/community/community/assets/22414094/c0dcacb0-9bd7-4d29-b3fd-f8ff30f8d991)
![image](https://github.com/community/community/assets/22414094/4e6649ab-29e5-4f22-bf38-900e9217ba4a)

**Comment by nderevjanik-fond** [comment]
I was able to work around this by pushing an empty commit (to the same branch as the out of sync PR):
> git commit --allow-empty -m "Forcing an empty commit."
> git push

**Reply by maniac-en** [reply]
This solution works like a `charm`, Thanks a ton! @nderevjanik-fond 🥇

**Reply by max-hontar** [reply]
Had already such issue 3 times for last 4 days. Another workaround - close & reopen PR.

**Reply by anastasialanz** [reply]
The empty commit worked for me as well.

**Reply by sheetalkadwal** [reply]
Thanks, it worked

**Reply by mike-loux-planview** [reply]
This has worked for me in the past, but not today.  The last time I saw this was a month or so ago, and at some point things suddenly magically updated and everything appeared without me doing anything.  Doesn't exactly inspire confidence.  
EDIT - And whaddaya know, I just tabbed back to that PR and there are the additional commits.  ***le sigh***

**Reply by rfilius** [reply]
still works !!

**Comment by buka4rill** [comment]
I closed the PR, opened it again, made a commit and pushed and it worked 🤷🏾

**Comment by lampwins** [comment]
Have also seen this happen where new commits not only don't show up in the view, they do not kick off CI. In this case, none of the above referenced work arounds work, however, changing the target branch of the PR and then changing it back seems to work.

**Reply by corquaid** [reply]
This is the case me also currently, changing the target branch to something else and then back again solves the issue.

**Reply by skeet70** [reply]
This worked for me as well, as did closing and re-opening the PR. Empty commit didn't do anything.

**Reply by dlyongemallo** [reply]
Going through the steps to change the *base* branch, without actually changing it (i.e., changing it to its current value), worked for me. 

Click "Edit" in the top right corner, then select the "base: " dropdown. Choose the existing value, and it will ask for a confirmation. Click "Change base", and this should trigger a process to update the PR (even though it's a noop).

**Reply by ampedro2003** [reply]
Worked perfectly for me! 👌

**Comment by huynguyennovem** [comment]
My case is PR view does not show even with a newly created PR. It displayed automatically before.

**Comment by JochenFromm** [comment]
In my case GitHub itself was apparently overloaded. The [GitHub Status page](https://www.githubstatus.com/) reported an incident and mentioned background job queues. In this case new commits do not appear immediately and it looks as if the Git repository is messed up, but it is only GitHub which is reacting much more slowly than usual. After a while the missing commits appeared.

**Comment by cikpak** [comment]
Just update the base to the same one and the issue will be fixed

**Reply by franco-mtz-skydropx** [reply]
i was going to post this solution It worked for me

**Reply by baxterjo** [reply]
Seeing this bug appear again today and this workaround fixed it.

**Reply by aarograh** [reply]
Also hit this today.  I just closed and reopened per a comment from a couple years ago, but good to know that setting the base might do the trick too.  Really really annoying.  Figuring this out took longer than the bug fix that was being merged!
